issuetracking/testdata/github_user.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Not a blocker but it might be good to append version number of GH api responses to these files